content/comply-starter-netlify-cms.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "Comply Netlify CMS Starter"
+github: https://github.com/ridem/comply-starter-netlify-cms
+author: ridem
+demo: https://comply-netlify-cms.netlify.com
+date: 2020-06-03
+ssg:
+    - Comply
```

Comment:
Comply will need to be added as a new ssg taxonomy

Reply:
Fixed